plugins/org.eclipse.reddeer.junit/src/org/eclipse/reddeer/junit/requirement/matcher/RequirementVersionMatcher.java

```diff
@@ -34,22 +34,17 @@
  * @author mlabuda@redhat.com
  *
  */
-public class RequirementVersionMatcher extends AbstractRequirementMatcher {
+public class RequirementVersionMatcher extends RequirementAttributeMatcher {
+	
+	private String matchingValue;
```

Comment:
String is very restrictive. Maybe i will want my matcher (that extends  RequirementAttributeMatcher) to accept a matcher value, not string value. What would I assign to matchingValue in RequirementAttributeMatcher? 

lets say i will implement
public class MyRequirementMatcher extends RequirementAttributeMatcher {
      MyRequirementMatcher(String attribute, Matcher valueMatcher){
    //what will be assigned to matchingValue in RequirementAttributeMatcher ??
}
}

On the other hand I can say that about attributeName RequirementAttributeMatcher too. So I will move attributeName and atributeValue to RequirementAttributeMatcher and change the type from String to Matcher.